### ISSUE TITLE
[cookbook] serverless blog suggestions for improvements

### ISSUE THREAD

**Issue by cedlemo** [vuejs/vuejs.org#1913]
First of all, thanks you for all these good documentations.

I did not found the cookbook entry "Create a CMS-Powered Blog" easy to follow. Maybe it is related to the fact that I am a beginner with Vuejs and with the JavaScript ecosystem.

For example in order to really start I needed to use the documentation of [ButterCMS](https://github.com/ButterCMS/buttercms-vue-tutorial).

Furthermore, in the recipe, it says that bulma is used in order to improve the appearance of the application and some screenshots are shown. But as I wanted to have the same pages that the one displayed in the screenshots, I had to go to the bulma documentation and try to use it myself.

Do you think that the problem is related to the fact that I was not enough skilled in Vuejs/JavaScript to try this recipe (it is ok) or that this recipe can be improved with more explanations on how to start and with better templates using bulma?

If you think that some improvements can be done, I am willing to send you a PR.

https://github.com/vuejs-fr/vuejs.org/issues/173

**Comment by sdras** [comment]
No, I don't think the issue is with you, I think the issue is with the recipe. I'm going to set aside time to go over some entries in the next few weeks and curate them. I'll either improve this recipe or get rid of it. Thank you for filing this.

**Comment by ricky11** [comment]
Would be good to have alternatives to buttercms, considering the price tag may be out of reach for most people. There are now much better i would say even free headless cms that could git in with Vue, some ideas are Prismic, Sanity, Vuepress?  I have not used any of these, but a alternative would be great..
